src/tv2_afvd_showstyle/parts/evs.ts

```diff
@@ -68,6 +69,7 @@ export async function CreatePartEVS(
 		metaData: {
 			type: Tv2PieceType.REPLAY,
 			outputLayer: Tv2OutputLayer.PROGRAM,
+			audioMode: partDefinition.sourceDefinition.vo ? Tv2AudioMode.VOICE_OVER : undefined,
```

Comment:
Shouldn't it be `Tv2AudioMode.FULL` when it is not voice over?

Reply:
Yes, it should

Reply:
I was unsure whether this mode should always be set, but i guess in this case, since it's either or, it makes perfect sense to do it. I will change it!

Reply:
Changed!

Reply:
If we can avoid it, we would prefer to avoid having `undefined` values :)